src/decisionengine/framework/modules/QueueLogger.py

```diff
@@ -0,0 +1,32 @@
+import multiprocessing
+
+from logging.handlers import QueueHandler, QueueListener
+
+
+class QueueLogger:
```

Comment:
A lot of the methods of this class seem to be missing test coverage.

Reply:
Right, I am working on this.

Reply:
I have added the test modules/tests/test_QueueLogger.py